Presentation.Web/Models/API/V2/Internal/Request/Notifications/UpdateScheduledNotificationWriteRequestDTO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Core.DomainModel;
+using Core.DomainModel.Notification;
+
+namespace Presentation.Web.Models.API.V2.Internal.Request.Notifications
+{
+    public class UpdateScheduledNotificationWriteRequestDTO: IHasBaseWriteProperties, IHasName, IHasToDate
+    {
+        /// <summary>
+        /// Name of the notification (different from the Subject)
+        /// </summary>
+        public string Name { get; set; }
```

Comment:
name is required right?

Reply:
It's not (at least in the UI) 

![image](https://user-images.githubusercontent.com/101557040/221834523-9a25b645-1587-4f98-846a-b11e05cdda94.png)